docs/book/src/user/troubleshooting.md

```diff
@@ -101,3 +101,26 @@ Alternatively a Cert Manager yaml file can be placed in the [clusterctl override
 
 More information on the clusterctl config file can be found at [its page in the book](../clusterctl/configuration.md#clusterctl-configuration-file)
 
+## Clusterctl failing to start providers due outdated image overrides
+
+clusterctl allows users to configure [image overrides](../clusterctl/configuration.md#image-overrides) via the clusterctl config file;
+however, when the image override is pinning providers image to a specific version, it could happen that this
+conflicts with clusterctl behavior of picking the latest version of a provider.
```

Comment:
```suggestion
clusterctl allows users to configure [image overrides](../clusterctl/configuration.md#image-overrides) via the clusterctl config file.
However, when the image override is pinning a providers image to a specific version, it could happen that this
conflicts with clusterctl behavior of picking the latest version of a provider.
```